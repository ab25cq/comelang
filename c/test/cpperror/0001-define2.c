/*
PATTERN: undef
PATTERN: c:6:
*/

#undef X

