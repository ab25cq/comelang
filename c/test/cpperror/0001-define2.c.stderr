cannot #undef macro that isn't defined at test/cpperror/0001-define2.c:6:8
#undef X
       ^
